virt/virt-web-console-overview.adoc

```diff
@@ -0,0 +1,274 @@
+
+.Migrations image:icon-folder.png[tab icon,25]
+[%collapsible]
+====
+* Time period image:icon-pencil.png[pencil icon,25]
+* VirtualMachineInstanceMigrations list image:icon-page.png[page icon,25]
+====
+
+[id='virtualization-overview-settings_{context}']
+=== Settings tab
+
+The *Settings* tab contains the following sections:
+
+* General image:icon-folder.png[tab icon,25]
+* Live migration image:icon-folder.png[tab icon,25]
+* Templates project image:icon-folder.png[tab icon,25]
+* User permissions image:icon-folder.png[tab icon,25]
+
+.Settings image:icon-folder.png[tab icon,25]
+[%collapsible]
+====
+
+[id='ui-virtualization-settings-general']
+.General image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* Service name
+* Provider
+* more settings
+=====
+
+[id='ui-virtualization-settings-live-migration']
+.Live migration image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* Max. migrations per cluster image:icon-pencil.png[pencil icon,25]
+* Max. migrations per node image:icon-pencil.png[pencil icon,25]
+* Live migration network image:icon-pencil.png[pencil icon,25]
+=====
+
+[id='ui-virtualization-settings-templates-project']
+.Templates project image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* Project  image:icon-pencil.png[pencil icon,25]
+=====
+
+[id='ui-virtualization-settings-user-permissions']
+.User permissions image:icon-folder.png[tab icon,25]
+[%collapsible]
+=====
+* List of user permissions
+=====
+
+====
+// end of Virtualization > Settings block
+
+[id='ui-virtualmachines-page_{context}']
+== VirtualMachines page
+
+The *VirtualMachines* page contains the following elements:
+
+* Virtualization dashboard image:icon-link.jpg[link icon,25]
+* Create VirtualMachine:
+
+  ** From form
+  ** From YAML
+
+* List of virtual machines
+
+Select and click a *VirtualMachine* to view the *VirtualMachine details* page.
+
+// Made this level 2 so that contents would appear in TOC
+[id='ui-virtualmachine-details-page_{context}']
+== VirtualMachine details page
+
+(Logically, this should be next level header. However, if we make this level 3, the tabs will not show up in the TOC. Suggestions or workarounds?)
+
+The *VirtualMachine details* page contains the following elements:
+
+* *Actions* menu
+
+** list of actions? or would that be too noisy? Could we arrange a list of actions horizontally, with comma, pipe, or slash delimiters?
+
+* xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-overview_{context}[Overview] image:icon-folder.png[tab icon,25]
+* xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-details_{context}[Details] image:icon-folder.png[tab icon,25]
+* xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-metrics_{context}[Metrics] image:icon-folder.png[tab icon,25]
+* xref:../virt/virt-web-console-overview.adoc#ui-virtualmachine-details-yaml_{context}[YAML] image:icon-folder.png[tab icon,25]
+
```

Comment:
There are 8 more tabs, if the env I'm logged into is up to date. I honestly wonder if a table would be better for these "...contains the following elements:" lists, to use some of the horizontal space.

Reply:
It's a ridiculous number of tabs! I will consider a table for the list of tabs, etc. I'm open to all possibilities. Thanks!